src/test/ui/consts/miri_unleashed/mutable_const.rs

```diff
@@ -12,9 +12,7 @@ const MUTABLE_BEHIND_RAW: *mut i32 = &UnsafeCell::new(42) as *const _ as *mut _;
 const MUTATING_BEHIND_RAW: () = {
     // Test that `MUTABLE_BEHIND_RAW` is actually immutable, by doing this at const time.
     unsafe {
-        *MUTABLE_BEHIND_RAW = 99 //~ WARN skipping const checks
-        //~^ ERROR any use of this value will cause an error
-        //~^^ tried to modify constant memory
+        *MUTABLE_BEHIND_RAW = 99 //~ ERROR constant contains unimplemented expression type
```

Comment:
Do we get the previous errors back if the feature gate is enabled on this file?

Reply:
no. We get this:
```
warning: skipping const checks
  --> /home/christian/Workspace/contrib/rust/src/test/ui/consts/miri_unleashed/mutable_const.rs:10:38
   |
LL | const MUTABLE_BEHIND_RAW: *mut i32 = &UnsafeCell::new(42) as *const _ as *mut _;
   |                                      ^^^^^^^^^^^^^^^^^^^^

error: any use of this value will cause an error
  --> /home/christian/Workspace/contrib/rust/src/test/ui/consts/miri_unleashed/mutable_const.rs:16:9
   |
LL | / const MUTATING_BEHIND_RAW: () = {
LL | |     // Test that `MUTABLE_BEHIND_RAW` is actually immutable, by doing this at const time.
LL | |     unsafe {
LL | |         *MUTABLE_BEHIND_RAW = 99 //~ ERROR constant contains unimplemented expression type
   | |         ^^^^^^^^^^^^^^^^^^^^^^^^ tried to modify constant memory
LL | |     }
LL | | };
   | |__-
   |
note: lint level defined here
  --> /home/christian/Workspace/contrib/rust/src/test/ui/consts/miri_unleashed/mutable_const.rs:5:9
   |
LL | #![deny(const_err)]
   |         ^^^^^^^^^

error: aborting due to previous error
```

Reply:
That's the previous error :D Please enable the feature gate on this file and revert to the preivous errors